lib/dashboard.js

```diff
@@ -107,8 +107,10 @@ app.get('/bundles/:bundleName/dashboard/*', ncgUtils.authCheck, (req, res, next)
 				if (err.code === 'ENOENT') {
 					return res.sendStatus(404);
 				}
-
-				return next();
+                
+                if (!res.headersSent) {
```

Comment:
It looks like this line is still indented with spaces, I wonder why eslint didn't catch that.

Reply:
Because it's actually fine (maybe lol). I used the GitHub editor to fix it quickly.

It looks wrong here: https://prnt.sc/j7mnlf
But when you open it for editing it's fine: https://prnt.sc/j7mnq3

So idk what this means. Quite odd